airbyte-integrations/connectors/destination-mongodb/src/main/java/io/airbyte/integrations/destination/mongodb/MongodbDestination.java

```diff
@@ -71,18 +78,35 @@ public AirbyteConnectionStatus check(final JsonNode config) {
     try {
       final var database = getDatabase(config);
       final var databaseName = config.get(JdbcUtils.DATABASE_KEY).asText();
-      final Set<String> databaseNames = MoreIterators.toSet(database.getDatabaseNames().iterator());
+      final Set<String> databaseNames = getDatabaseNames(database);
       if (!databaseNames.contains(databaseName) && !databaseName.equals(database.getName())) {
         throw new MongodbDatabaseException(databaseName);
       }
       return new AirbyteConnectionStatus().withStatus(AirbyteConnectionStatus.Status.SUCCEEDED);
+    } catch (final ConnectionErrorException e) {
+      var messages = getErrorMessage(e.getStateCode(), e.getErrorCode(), e.getExceptionMessage(), e);
```

Comment:
Yes, that's right, it's possible. But in processing, in the `ErrorMessage.getErrorMessage` method, I'll process it, and if the error code is empty, it won't change the readability of the processed error message.

Reply:
Wouldn't that mean that you can be passing in an `uninitialized` int into `getErrorMessage`? That would cause an error cause in `MongoDbSource` you create a `new ConnectionErrorException(String exceptionMessage)` but that does not include an `errorCode`

When I tested this quick snipped out in my IDE this was giving my an error compiling which mimics how `getErrorMessage` works, correct me if I'm wrong
```java
  @Test
  void testUninitialzedIntPassed() {
    final int number;
    System.out.println(runTest("hi", number, "world"));
  }

  public String runTest(final String first, final int num, final String second) {
    final String temp = num == 0 ? "" : " TEST";
    return first + temp + second;
  };
```